solr/core/src/java/org/apache/solr/highlight/SolrHighlighter.java

```diff
@@ -91,13 +87,25 @@ protected boolean emptyArray(String[] arr) {
     return (arr == null || arr.length == 0 || arr[0] == null || arr[0].trim().length() == 0);
   }
 
+  private static String[] expandWildcardsInHighlightFields(
```

Comment:
#757 would later require this to be `protected` instead of `private` visibility.